src/render_cmd.rs

```diff
@@ -198,7 +203,7 @@ impl RenderCmd {
                     cmd.join(" ")
                 );
 
-                self.run_helm(&cmd.join(" "), std::io::sink())?;
+                self.run_helm(&cmd.join(" "), &std::io::sink())?;
```

Comment:
The added `&` shouldn't be necessary.
```suggestion
                self.run_helm(&cmd.join(" "), std::io::sink())?;
```